Streamline dev setup docs (#99)
thehereward/snapcup: docs/Firebase.md

# Firebase info

## Installation

To install the firebase cli and emulator you must have npm installed already and run:

```bash
sudo npm install -g firebase-tools
```

> To verify this has worked try running the `firebase` command in your terminal (may need a restart).

Reviewer: ```suggestion
To verify this has worked try running the `firebase` command in your terminal (you may to restart your terminal).
```

Unless you meant a _machine_ restart.

Author: No it is a terminal restart good spot

## Enabling complex sorting and filtering on Firestore

> In order to be able to chain the filter and sort needed to get a user's snaps and sort by date, navigate to your firestore console and add a index with collection id `snaps`, fields indexed `from: Ascending` `timestamp: Descending` and query scope `Collection`.

Reviewer: I _think_ this should be unnecessary now (the emulator is clever enough to persist local state).

Author: Shall I remove?

Author: Yep just checked that the `snaps` collection gets created automatically in firestore

## Installation

> To install the firebase cli and emulator you must have npm installed alreadynpm run and run:

Reviewer: A find and replace error?

Author: Don't know how that got in there but good spot

Author: Corrected
